src/goto-analyzer/unreachable_instructions.cpp

```diff
@@ -289,9 +289,10 @@ static void xml_output_function(
   x.set_attribute("last line", id2string(last_location.get_line()));
 }
 
+template <class T>
 static void list_functions(
   const goto_modelt &goto_model,
-  const std::set<irep_idt> called,
+  const T called,
```

Comment:
I'm aware this will cause more changes, but I'd insist that the template is unnecessary here and everyone should be using the updated interface.